css/_project-custom.scss

```diff
@@ -1862,3 +1836,24 @@ li {
     @include u-margin-top(1);
   }
 }
+
+p + .usa-accordion,
+h2 + *,
+h3 + *,
+h4 + *,
+h5 + *,
+h6 + * {
+  margin-top: units(2);
+}
+
+.usa-sidenav a {
+  padding-right: units(0.5);
```

Comment:
What is this code used for?

Reply:
For reducing the right padding on the nav items, since it felt like there was too much — it was causing some line breaks that didn't feel strictly necessary.

Reply:
For the site sidenav or mobile nav or both? Could you scope it to the site elements and not the sidenav preview?

Reply:
What's the best way to scope to site here?

Reply:
Wonder if there's a `:not()` selector solution here that could also help us think about our prose scope bleed into component previews.